client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
Do you mean to keep it visible and active even if using previewMarkup? Or
visible and disabled? I think it's better to hide it if the button doesn't
do anything.

On Mon, Mar 21, 2016 at 6:45 AM Matias Ventura <notifications@github.com>
wrote:

> In client/components/web-preview/index.jsx
> <https://github.com/Automattic/wp-calypso/pull/4136#discussion_r56804241>:
>
> > @@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
> >  	getDefaultProps() {
> >  		return {
> >  			showExternal: true,
>
> Mmm, I think we should keep the external button here because it's
> important for it to be in the same place anytime we preview something.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/wp-calypso/pull/4136/files/29b9995fa6b5b12855ed4742bce30bf157966b6a#r56804241>
>